src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [OPTIONAL] It will be nice to update the value of action type to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:
```
const CHECK_STATUS = bookstore/categories/CHECK_STATUS
```

Reply:
I totally forgot thanks alot.